docs/examples.md

```diff
@@ -23,6 +23,43 @@ Some examples include multiple classes in a single source file. This
 is purely to make the examples easier to follow when browsing on
 GitHub.
 
+## Running and deploying
+
+Each example can be run locally from the command line by changing to
+the example directory and running
+
+```sh
+dotnet run
+```
+
+Alternatively, you can open the solution file in the `examples`
+directory, set the "Startup Project" to the example you want to run,
+and just press F5 to run.
+
+To deploy to Google Cloud Functions or Cloud Run, follow the
+[deployment guide](deployment.md). Every example uses a function
+class called `Function` in a namespace that's the same as the
+directory. For example, the entry point for the function in the
+`Google.Cloud.Functions.Examples.SimpleHttpFuntion` directory is
+`Google.Cloud.Functions.Examples.SimpleHttpFunction.Function`.
+
+A sample command line is given for each example, but you may need to
+adapt this to specify your own Google Cloud resources such as PubSub
```

Comment:
Pub/Sub

Reply:
Done (here and elsewhere). I'll do a quick grep to see if I've got this elsewhere other than identifiers - but I'll make changes to other files in a new PR.